release/ray_release/cluster_manager/cluster_manager.py

```diff
@@ -109,7 +109,7 @@ def _annotate_cluster_compute(
 
         cluster_compute = cluster_compute.copy()
         aws = cluster_compute.get("aws", {})
-        cluster_compute["aws"] = add_tags_to_aws_config(
+        cluster_compute["advanced_configurations_json"] = add_tags_to_aws_config(
```

Comment:
lgtm. If this runs in staging, it should be ok since staging is already deployed